### PR TITLE
Cep 440 campaign bug fix

### PR DESCRIPTION
**Pull request by StevenChin**
[Jira Issue](https://medullan.atlassian.net/browse/CEP-440)

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ class AppKernel extends Kernel
      *
      * @const string
      */
-    const EXTRA_VERSION = '';
+    const EXTRA_VERSION = '-2.0.1-alpha';
```
Comment (StevenChin) [inline app/AppKernel.php]: The plan is to bump this version to `-2.0.1` when we publish the release from master. Alternatively we can just remove the version changes.

Comment (StevenChin) [reply app/AppKernel.php]: Based on recent discussions, these changed will be removed.

**Comment by StevenChin** [comment]
Closing this PR as we decided to downgrade to version `2.15.2`

**Comment by kuetemeier** [comment]
Hi StevenChin, I just tested your bug fix with mautic/mautic 2.15.3 and it works great for me. Could you please send an official pull request to the mautic source code repo, so the fix gets integrated into the main source repo?

**Comment by StevenChin** [comment]
Hey @kuetemeier, this is actually an open pull request to Mautic, https://github.com/mautic/mautic/pull/7704, that we merged. It seems to been earmarked for the next release but we needed it asap.

**Comment by kuetemeier** [comment]
I see, thx. Yes I needed it asap, too. Thanks for the quick response.

**Comment by StevenChin** [comment]
In that case, as a quick fix you can apply the patch manually to your Mautic instance, provided that you have SSH access to the server. Check out https://forum.mautic.org/t/contacts-not-running-through-campaign-again-although-restart-the-campaign-is-set/11561/5 for how to apply the patch.

**Comment by kuetemeier** [comment]
I have already done it (found this merge request via google). It's working great, thank you.

**Comment by StevenChin** [comment]
Great! Glad I could help.

**Pull request closed without merging**